ui/src/app/features/user-table/user-table.component.ts

```diff
@@ -98,4 +100,13 @@ export class UserTableComponent extends TableComponent implements OnInit, OnChan
       this.message = this.createMessage;
     }
   }
+
+  refreshPage(): void {
+    if (!localStorage.getItem('refresh-page')) {
+      localStorage.setItem('refresh-page', 'no reload');
```

Comment:
will this reload the page every time user is navigated to user table? There should be more appropriate fix than this

Reply:
okay i will fix them tonight